### ISSUE TITLE
Jam with the Band (Europe) doesn't save with NTR clock speed

### ISSUE THREAD

**Issue by Agusum** [DS-Homebrew/nds-bootstrap#1622]
**Your console:** New Nintendo 3DS XL

**Launch Method:** SD

**Version used:** Release v1.0.0

**Game tested/used/applicable:** Jam with the Band (Europe)

**Save file:**

**Expected Behavior:** The game should save normally

**Actual Behavior:** Constantly fails to save the game, impeding proceeding to the game

### Steps to reproduce
1. Start the game
2. Fill in your artist profile
3. Fail to save

### nds-bootstrap.ini
```[NDS-BOOTSTRAP]
DEBUG = 0
LOGGING = 0
CACHE_FAT_TABLE = 0
ROMREAD_LED = 0
DMA_ROMREAD_LED = -1
CONSOLE_MODEL = 2
NDS_PATH = sd:/roms/nds/Jam with the Band (Europe) (En,Fr,De,Es,It).nds
SAV_PATH = sd:/roms/nds/saves/Jam with the Band (Europe) (En,Fr,De,Es,It).sav
RAM_DRIVE_PATH = sd:/null.img
GUI_LANGUAGE = en
LANGUAGE = -1
REGION = -1
DSI_MODE = 1
BOOST_CPU = 0
BOOST_VRAM = 0
CARD_READ_DMA = 1
ASYNC_CARD_READ = 0
EXTENDED_MEMORY = 0
DONOR_SDK_VER = 0
PATCH_MPU_REGION = 0
PATCH_MPU_SIZE = 0
FORCE_SLEEP_PATCH = 0
HOMEBREW_ARG =
PRECISE_VOLUME_CONTROL = 0
SDNAND = 0
MACRO_MODE = 0
SOUND_FREQ = 0
HOTKEY = 284
AP_FIX_PATH = sd:/_nds/nds-bootstrap/apFix.ips
USE_ROM_REGION = 1
B4DS_MODE = 0
```

### Other notes
One time I was able to get past the first time the game saves but it failed the second time the game tried to do it after the initial tutorial.

### Will you be available for future issues and/or replies?
Yes

![IMG_3136](https://github.com/DS-Homebrew/nds-bootstrap/assets/4369698/9ef1d223-4789-4182-8c62-84f453f23c65)

**Comment by RocketRobz** [comment]
I have no idea what's causing the error to occur sometimes. Have you tried running it with TWL clock speed?

**Comment by Agusum** [comment]
That seems to do the trick, thanks!

1. When selecting TWL clock speed, does it matter the Run in option that I choose or will it default to DSi mode when in Auto?
2. Is it possible to patch this clock speed in nds-bootstrap so that users don't need to fiddle with the setting?

**Comment by RocketRobz** [comment]
> That seems to do the trick, thanks!
> 
> 1. When selecting TWL clock speed, does it matter the Run in option that I choose or will it default to DSi mode when in Auto?
> 2. Is it possible to patch this clock speed in nds-bootstrap so that users don't need to fiddle with the setting?

1. It does not matter what `Run in` is set to.
2. For a while, it has worked fine without TWL clock speed, so idk why it needs it now. If more people report the same issue, then it might be looked into.

**Comment by bibarub** [comment]
happens on my DSi as well (at least with NTR clock speed). though not as frequently

**Comment by Nicola653** [comment]
I know the problem, is bc the game dont use a chip to save but use the rom to save, so you need to start the game in a emulator on a mobile or pc and then put the .sav file in your 3ds sd card

**Comment by RocketRobz** [comment]
> I know the problem, is bc the game dont use a chip to save but use the rom to save, so you need to start the game in a emulator on a mobile or pc and then put the .sav file in your 3ds sd card

nds-bootstrap handles the save type for that game just fine, so that's not why it fails to save.

**Comment by Nicola653** [comment]
> > Conosco il problema, è perché il gioco non usa un chip per salvare ma usa la rom per salvare, quindi devi avviare il gioco in un emulatore su un cellulare o un PC e quindi inserire il file .sav nella scheda SD 3ds
> 
> nds-bootstrap gestisce perfettamente il tipo di salvataggio per quel gioco, quindi non è questo il motivo per cui non riesce a salvare.

I dont think so, because even my 3ds when i lauched the rom from the home menu shortcut when it goes the saving game moment it gives this error EVERY TIME.
But when i start the game and save whit a phone emulator it saves, then i put the save file into the sd card seve folder and you start the game from the point the emulator saved,

**Comment by RocketRobz** [comment]
> > > Conosco il problema, è perché il gioco non usa un chip per salvare ma usa la rom per salvare, quindi devi avviare il gioco in un emulatore su un cellulare o un PC e quindi inserire il file .sav nella scheda SD 3ds
> > 
> > 
> > nds-bootstrap gestisce perfettamente il tipo di salvataggio per quel gioco, quindi non è questo il motivo per cui non riesce a salvare.
> 
> I dont think so, because even my 3ds when i lauched the rom from the home menu shortcut when it goes the saving game moment it gives this error EVERY TIME. But when i start the game and save whit a phone emulator it saves, then i put the save file into the sd card seve folder and you start the game from the point the emulator saved,

You need to switch to TWL clock speed in order for the game to save. To do so with a forwarder, launch it, then hold the Y button both before and after the white screens appear, and change the ARM9 CPU speed option.

**Comment by RocketRobz** [comment]
Seems to be fixed as of v1.6.2.
